.github/workflows/tests.yaml

```diff
@@ -1,22 +0,0 @@
-name: Unit tests
```

Comment:
The forkflow now uses step-by-step validation in a different file:

https://github.com/diplodoc-platform/html-extension/commit/c72d01a4591461895653fc529efb916a4c9a1e90#diff-d7a14ffe16d26e1f54f1e3f02a8ebf52f96fae5098e252044ee682e03c3928b3